libcxx/test/libcxx/ranges/range.adaptors/range.move.wrap/no_unique_address.pass.cpp

```diff
@@ -8,49 +8,73 @@
 
```

Comment:
I might suggest keeping the old test around, but renaming it to something like `empty_object.pass.cpp`, since it seems to have been written specifically to test something about empty objects. I don't remember why I wrote it that way anymore, and https://reviews.llvm.org/D102135 is not responsive right now but I suspect there might be a rationale.

I would also rename copyable-box to movable-box as a drive-by in that test file, since it was forgotten when we renamed the type.